recipes/ccphylo/LICENSE

```diff
@@ -0,0 +1,18 @@
+/* Philip T.L.C. Clausen Jul 2022 plan@dtu.dk */
```

Comment:
It seems you included the license twice.

Reply:
(since you're one of the authors if I am not mistaken, you could also consider to add the license file to the repository; just as a suggestion.)